tests/PSRule.Rules.Azure.Tests/Resources.MySQL.json

```
@@ -451,6 +464,19 @@
         "Type": "Microsoft.DBforMySQL/flexibleServers/administrators",
         "ResourceType": "Microsoft.DBforMySQL/flexibleServers/administrators",
         "SubscriptionId": "00000000-0000-0000-0000-000000000000"
+      },
+      {
+        "ResourceName": "aad_auth_only",
+        "Name": "aad_auth_only",
+        "Properties": {
+          "value": "ON",
+          "currentValue": "ON",
```

Test if the current value is being overridden.

```suggestion
          "currentValue": "OFF",
```